crates/web-sys/webidls/enabled/Window.webidl

```diff
@@ -25,7 +25,9 @@
 // interface XULControllers;
 
 // http://www.whatwg.org/specs/web-apps/current-work/
-[PrimaryGlobal, LegacyUnenumerableNamedProperties, NeedResolve]
```

Comment:
Would it be possible to recognize `PrimaryGlobal` instead of adding the modifications here perhaps?

Reply:
`PrimaryGlobal` was removed from spec ([old](https://www.w3.org/TR/WebIDL-1/#PrimaryGlobal), [new](https://heycam.github.io/webidl/#Global)), I have just applied this commit: https://github.com/whatwg/html/commit/8c69161a98acea9eeef1e43e481ca3cc6287d393